lite/configure.ac

```diff
@@ -138,12 +138,18 @@ dnl https://github.com/libxmp/libxmp/issues/385
    if test $ac_cv_c_flag_w_all = yes; then
       CFLAGS="${CFLAGS} -Werror"
    fi
-   XMP_TRY_COMPILE(whether compiler understands externally_visible attribute,
+   XMP_TRY_COMPILE(whether compiler understands attribute externally_visible,
```

Comment:
This seems unrelated, but I guess this was done for consistency of `configure` output. I'd make it a separate commit, though.

Reply:
removed that cosmetic change for now, will push when the buildbots complete.